src/IR.h

```diff
@@ -651,6 +651,19 @@ struct Call : public ExprNode<Call> {
         return is_intrinsic() && this->name == get_intrinsic_name(op);
     }
 
+    bool is_intrinsic(std::initializer_list<IntrinsicOp> intrinsics) const {
```

Comment:
What's this about? Isn't this given by the call type?

Reply:
This is the same as is_intrinsic(IntrinsicOp), except it allows checking for multiple intrinsics at a time a bit more easily

Reply:
Oh right, I was misinterpreting the arg as being the list of all intrinsics that exist